R/tmb-methods.R

```diff
@@ -207,3 +207,111 @@ print.mmrm_tmb <- function(x,
   cat("\n")
   invisible(x)
 }
+
+
+#' Residuals method for mmrm objects
+#'
+#' @describeIn mmrm_tmb_method to obtain residuals - either unscaled or normalized
+#' @param object (`mmrm_tmb`)\cr the fitted MMRM.
+#' @param type (`character`)\cr unscaled ('response'), 'pearson' or 'normalized'. Default is 'response',
```

Comment:
```suggestion
#' @param type (`string`)\cr unscaled (`response`), `pearson` or `normalized`. Default is `response`,
```
since needs to be scalar.